PWGHF/TableProducer/candidateSelectorDplusToPiKPi.cxx

```diff
@@ -216,13 +258,62 @@ struct HfCandidateSelectorDplusToPiKPi {
 
       if (!selectionPID(pidTrackPos1Pion, pidTrackNegKaon, pidTrackPos2Pion)) { // exclude D±
         hfSelDplusToPiKPiCandidate(statusDplusToPiKPi);
+        if (applyML) {
+          hfMlDplusToPiKPiCandidate(outputML);
+        }
         continue;
       }
       SETBIT(statusDplusToPiKPi, aod::SelectionStep::RecoPID);
       if (activateQA) {
         registry.fill(HIST("hSelections"), 2 + aod::SelectionStep::RecoPID, ptCand);
       }
 
+      if (applyML) {
+        // tracks without TPC nor TOF already rejected at PID selection level
+        // tag tracks with TPC and TOF, only TPC, only TOF
+        auto tagPIDPos1 = o2::analysis::tagPID(trackPos1);
+        auto tagPIDNeg = o2::analysis::tagPID(trackNeg);
+        auto tagPIDPos2 = o2::analysis::tagPID(trackPos2);
+
+        auto combinedNSigmaPiPos1 = getCombinedNSigma(trackPos1.tpcNSigmaPi(), trackPos1.tofNSigmaPi(), tagPIDPos1);
```

Comment:
How come you don't need `o2::analysis::getCombinedNSigma` as you do for `tagPID`?
I don't see `using namespace o2::analysis`.

Reply:
It woked without it but this is strange indeed. I added the prefix.